### PR TITLE
What is CI/CD what is page

### PR DESCRIPTION
**Pull request by interurban**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/what-is/what-is-ci-cd.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### What is a CI/CD pipeline?
+
+The sequential nature of continuous integration, delivery, and deployment practices creates a symbolic pipeline, hence the CI/CD pipeline.
+
+A pipeline is the deployable unit path for CI/CD. A pipeline starts when code is committed to a repository like GitHub. The next step is a notification to a build system, such as [GitHub Actions](https://docs.github.com/en/actions). The build system compiles the code and runs unit tests. Integration tests are the next step if your code passes the unit tests. If your code passes both unit and integration tests, the images will be created and pushed into a registry service in the case of containers. This is the simplest example of a pipeline. Still, you can do many more things, such as security scans, check modules for CVEs (Common Vulnerabilities and Exploits), send Slack notifications, and run quality checks. A pipeline can be fully automated or have checkpoints that require approval before resuming.
+
+### CI/CD terms and tools
```
Comment (desteves) [inline themes/default/content/what-is/what-is-ci-cd.md:45]: This table is super helpful!

File: themes/default/content/what-is/what-is-ci-cd.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+| Feature Flagging | A technique that allows developers to turn certain functionalities on and off, typically used in a continuous delivery environment. | [LaunchDarkly](https://launchdarkly.com/), [FeatureToggle](https://featuretoggle.org/) |
+| Test Automation | Using special software to control the execution of tests and the comparison of actual outcomes with predicted outcomes. | [Selenium](https://www.selenium.dev/), [JUnit](https://junit.org/junit5/) |
+| Version Control Systems (VCS) | Systems that record changes to a file or set of files over time so that you can recall specific versions later. | [Git](https://git-scm.com/), [Subversion](https://subversion.apache.org/) |
+
+## Best practices for CI/CD pipelines
+
+To build an effective CI/CD pipeline, platform and development teams must adopt and invest in DevOps best practices.
+
+* **Use a single repository** - Maintain a centralized repository containing all the necessary files, scripts, source code, and resources for the build and deployment processes.
+* **Adopt [link](link)**  - Integrate code changes regularly into the main branch to avoid complex merges and ensure a continuous flow of minor, incremental updates.
```
Comment (desteves) [inline themes/default/content/what-is/what-is-ci-cd.md:65]: Hmm unsure what happened here with the link?

**Review by desteves — APPROVED** [review]
I left very minor comments but approving the PR so it's not blocked. LGTM

File: themes/default/content/what-is/what-is-ci-cd.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+title: What is Continuous Integration/Continuous Delivery (CI/CD)?
+meta_desc: |
+    Learn about CI/CD practices that improve dev process with automation for effective, rapid software delivery.
+type: what-is
+page_title: "What is Continuous Integration/Continuous delivery (CI/CD)?"
```
Comment (interurban) [inline themes/default/content/what-is/what-is-ci-cd.md:6]:
```suggestion
page_title: "What is Continuous Integration/Continuous Delivery (CI/CD)?"
```

File: themes/default/content/what-is/what-is-ci-cd.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+| Continuous delivery (CD) | A software engineering approach in which teams produce software in short cycles, ensuring that the software can be reliably released at any time. It aims at building, testing, and releasing software with greater speed and frequency. | [Codefresh](https://codefresh.io/docs/docs/getting-started/cd-codefresh/), [AWS CodePipeline](https://aws.amazon.com/codepipeline/), [GitHub Actions](https://github.com/features/actions) |
+| Continuous integration (CI) | A development practice where developers integrate code into a shared repository frequently, ideally several times a day. Each integration is verified by an automated build, allowing teams to detect problems early. | [Jenkins](https://www.jenkins.io/), [CircleCI](https://circleci.com/) |
+| Continuous testing | The process of executing automated tests as part of the software delivery pipeline to obtain immediate feedback on the business risks associated with a software release candidate. | [Selenium](https://www.selenium.dev/), [Appium](https://appium.io/docs/) |
```
Comment (interurban) [inline themes/default/content/what-is/what-is-ci-cd.md:55]:
```suggestion
| Continuous testing | The process of executing automated tests as part of the software delivery pipeline to obtain immediate feedback on the business risks associated with a software release candidate. | [Selenium](https://www.selenium.dev/), [Appium](https://appium.io/docs/en/2.2/) |
```

File: themes/default/content/what-is/what-is-ci-cd.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+| Continuous integration (CI) | A development practice where developers integrate code into a shared repository frequently, ideally several times a day. Each integration is verified by an automated build, allowing teams to detect problems early. | [Jenkins](https://www.jenkins.io/), [CircleCI](https://circleci.com/) |
+| Continuous testing | The process of executing automated tests as part of the software delivery pipeline to obtain immediate feedback on the business risks associated with a software release candidate. | [Selenium](https://www.selenium.dev/), [Appium](https://appium.io/docs/en/2.2/) |
+| Feature Flagging | A technique that allows developers to turn certain functionalities on and off, typically used in a continuous delivery environment. | [LaunchDarkly](https://launchdarkly.com/), [FeatureToggle](https://featuretoggle.org/) |
```
Comment (interurban) [inline themes/default/content/what-is/what-is-ci-cd.md:56]:
```suggestion
| Feature flagging | A technique that allows developers to turn certain functionalities on and off, typically used in a continuous delivery environment. | [LaunchDarkly](https://launchdarkly.com/), [FeatureToggle](https://featuretoggle.org/) |
```

File: themes/default/content/what-is/what-is-ci-cd.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+| Continuous integration (CI) | A development practice where developers integrate code into a shared repository frequently, ideally several times a day. Each integration is verified by an automated build, allowing teams to detect problems early. | [Jenkins](https://www.jenkins.io/), [CircleCI](https://circleci.com/) |
+| Continuous testing | The process of executing automated tests as part of the software delivery pipeline to obtain immediate feedback on the business risks associated with a software release candidate. | [Selenium](https://www.selenium.dev/), [Appium](https://appium.io/docs/en/2.2/) |
+| Feature flagging | A technique that allows developers to turn certain functionalities on and off, typically used in a continuous delivery environment. | [LaunchDarkly](https://launchdarkly.com/), [FeatureToggle](https://featuretoggle.org/) |
+| Test Automation | Using special software to control the execution of tests and the comparison of actual outcomes with predicted outcomes. | [Selenium](https://www.selenium.dev/), [JUnit](https://junit.org/junit5/) |
```
Comment (interurban) [inline themes/default/content/what-is/what-is-ci-cd.md:57]:
```suggestion
| Test automation | Using special software to control the execution of tests and the comparison of actual outcomes with predicted outcomes. | [Selenium](https://www.selenium.dev/), [JUnit](https://junit.org/junit5/) |
```

**Pull request merged**